app/components/secure/WalletSecureComponent.tsx

```diff
@@ -156,27 +118,34 @@ export const WalletSecureFragment = systemFragment((props: { mnemonics: string,
             break;
     }
 
-    const disabled = props.deviceEncryption === 'none';
-    if (disabled) {
-        text = t('secure.subtitleNoBiometrics');
-        buttonText = t('secure.titleUnprotected');
-        icon = undefined;
-    }
-
-    const continueAnywayAlert = React.useCallback(() => {
+    const onLater = React.useCallback(() => {
         Alert.alert(
-            t('secure.titleUnprotected'),
-            t('secure.messageNoBiometrics'),
+            t('secure.onLaterTitle'),
+            t('secure.onLaterMessage'),
             [
                 { text: t('common.cancel') },
                 {
-                    text: t('common.continueAnyway'), onPress: () => {
-                        onClick(true);
-                    }
+                    text: t('secure.onLaterButton'), onPress: props.onLater
                 }
-            ])
+            ]
+        );
     }, []);
 
+    if (disableEncryption) {
+        if (props.import) {
+            const address = React.useMemo(() => getBackup(), []);
```

Comment:
Conditional hooks are prohibited in React. https://legacy.reactjs.org/docs/hooks-rules.html

Reply:
Yeah, I might have missed this when I was just reusing old code, fixed.